mod-gobi/examples/orders_mapping.sample

```diff
@@ -53,13 +45,6 @@
             "default": "USD"
           }
         },
-        {
-          "field": "DATE_ORDERED",
-          "dataSource": {
-            "from": "//OrderPlaced",
-            "translation": "toDate"
-          }
-        },
         {
```

Comment:
BILL_TO, SHIP_TO, LICENSE_CODE also need to be deleted from sample file, because they deleted from mapping.json file

Reply:
sure, done.